package.json

```
@@ -48,7 +48,7 @@
     "themes-generate": "esbuild --target=es6 ./scripts/cli/generateSassVariableFiles.ts --bundle --platform=node --tsconfig=./scripts/cli/tsconfig.json | node",
     "themes:usage": "eslint . --ignore-pattern '*.test.ts*' --ignore-pattern '*.spec.ts*' --cache --plugin '@grafana' --rule '{ @grafana/theme-token-usage: \"error\" }'",
     "typecheck": "tsc --noEmit && yarn run packages:typecheck",
-    "plugins:build-bundled": "find plugins-bundled -name package.json -not -path '*/node_modules/*' -execdir yarn build \\;",
+    "plugins:build-bundled": "",
```

This can't be fully removed yet due to https://github.com/grafana/grafana-build/blob/842fd595ef1f2671bc4443b754adc215822887e6/frontend/build.go#L19

---

curious, can't that be removed first? If that's not the case, this could be something like 

```suggestion
    "plugins:build-bundled": "echo 'bundled plugins is no longer supported'",
```

---

The only reason I can think of for not removing it now is the grafana-build repo needs to be backwards compatible with older versions of Grafana. I'm not sure how far back it needs to go but I'd vote for echoing right now. 👍

---

Yeah I was planning on looping in a grafana-build wizard such as @kminehart to check what the best approach is here